### PR TITLE
N8 n 4156 stickies

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

File: packages/editor-ui/src/views/NodeView.vue [hunk 1]
```diff
@@ -1370,6 +1373,22 @@ export default mixins(
...
+			addStickyNote() {
+				if (document.activeElement) {
+					(document.activeElement as HTMLElement).blur();
+				}
+
+				const offset: [number, number] = [...(this.$store.getters.getNodeViewOffsetPosition as [number, number])];
+
+				const position = CanvasHelpers.getMidCanvasPosition(this.nodeViewScale, offset);
+				position[0] -= DEFAULT_STICKY_WIDTH / 2;
+				position[1] -= DEFAULT_STICKY_HEIGHT / 2;
```
Comment (alexgrozav) [inline packages/editor-ui/src/views/NodeView.vue:1385]: New stickies are not getting added to the center for me. You probably have to add the `SIDEBAR_WIDTH` and `HEADER_HEIGHT` into the offset calculation here somewhere (?).

See screenshot below, the position with respect to the execute button.

image

**Review by alexgrozav — COMMENTED** [review]
Great work! Only one minor issue.

File: packages/editor-ui/src/views/NodeView.vue (hunk 1 above)
Comment (mutdmour) [reply packages/editor-ui/src/views/NodeView.vue:1385]: I don't think it has to be perfect. will double check with David

Comment (mutdmour) [reply packages/editor-ui/src/views/NodeView.vue:1385]: checked with David. this is good. sidebar width is accounted for actually

Comment (mutdmour) [reply packages/editor-ui/src/views/NodeView.vue:1385]: it's weird. but not worth the effort

**Review by alexgrozav — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.188.0`

**Pull request merged**